models/category.ts

```diff
@@ -0,0 +1,68 @@
+import { Category } from "../types/categories.js";
+
+export class CategoryRepo {
+  categories: Category[] = [
+    {
```

Comment:
Doesn't look like categories names. It's better to change)

Reply:
Thanks, Done